src/base-command.ts

```diff
@@ -16,6 +20,15 @@ export default abstract class extends Command {
     help: flags.help({ char: 'h' }),
   };
 
+  checkFlagDeprecations(flags: any, flagDefinitions: any) {
```

Comment:
Something I didn't cover yet, but our coding style is:

snakecase for vars
camelcase for functions

Reply:
oh thanks- I forgot to ask about this earlier. I did another quick review and I think this function had the only styling violations. Should be fixed in the latest commit.